test/ebaze_web/controllers/session_controller_test.exs

```diff
@@ -19,5 +19,13 @@ defmodule EbazeWeb.SessionControllerTest do
       assert get_flash(conn, :info) =~ "welcome back!"
       assert redirected_to(conn) == Routes.page_path(conn, :index)
     end
+
+    test "POST /sign-in index page displays option to add 'new listing'", %{conn: conn} do
+      Accounts.create_user(@user_attrs)
```

Comment:
maybe make a helper test method in test helper for this? Signing in?

Reply:
https://trello.com/c/YJmuQDoq